src/Fixer/ControlStructure/NoUnneededControlParenthesesFixer.php

```diff
@@ -183,6 +179,33 @@ public function isCandidate(Tokens $tokens): bool
      */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     {
+        if (0 === \count($this->noopTypes)) {
```

Comment:
- should we consider [`empty`](https://3v4l.org/eQOke/vld) over [`===` and `count`](https://3v4l.org/XAve1/vld) ? one VLD fewer ;)

I wound't call it each time we applyFix. can we move to constructor?

Reply:
and then mark the property as `@readonly`

Reply:
`empty($var)` is the same as `!isset($var) || !$var`. Basically, a way to make a loose comparison look as if it's strict.

Reply:
BTW, a loose comparison has even fewer VLDs: https://3v4l.org/Ung31/vld.

Reply:
one more reason to not have this condition at all

Reply:
I pick explicit over loose comparison any day, but happy to refactor